connector/connector.go

```diff
@@ -65,18 +65,24 @@ type CallbackConnector interface {
 	HandleCallback(s Scopes, r *http.Request) (identity Identity, err error)
 }
 
+// SAMLConnector bindings
+const (
+	SAMLBindingPOST     = "post"
```

Comment:
I meant the "post" and "redirect" values.

Actually, I think the way it currently is is fine. Let's keep it as is.